rune/libenclave/attestation/sgx/attest.go

```diff
@@ -6,3 +6,7 @@ const (
 	EPID
```

Comment:
Please rename this go file to attestation.go

Reply:
> Please rename this go file to attestation.go

This is already fixed.